scripts/ops/add_meta_module.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ -z "$1" ]; then
```

Comment:
Though: would verify if `jq` and `fedimint-cli` are in the path, before starting to do anything.

Reply:
done